src/utils/HOCs/withIris.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from 'react';
 import { ThemeContext } from 'styled-components';
 
-const version = '0.143.1';
```

Comment:
The next version should be bumped to `0.144.0` since features were added to the sidebar minors.